src/helm/benchmark/presentation/run_specs_dec2023.conf

```diff
@@ -0,0 +1,169 @@
+  {description: "math:model=text_code,subject=counting_and_probability,level=3,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=3,use_chain_of_thought=True", priority: 3}
+
+  {description: "math:model=text_code,subject=number_theory,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=algebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=prealgebra,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=geometry,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=counting_and_probability,level=4,use_chain_of_thought=True", priority: 4}
+  {description: "math:model=text_code,subject=precalculus,level=4,use_chain_of_thought=True", priority: 4}
+
+  {description: "math:model=text_code,subject=number_theory,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=intermediate_algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=algebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=prealgebra,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=geometry,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=counting_and_probability,level=5,use_chain_of_thought=True", priority: 3}
+  {description: "math:model=text_code,subject=precalculus,level=5,use_chain_of_thought=True", priority: 3}
+
+  # GSM
+  {description: "gsm:model=text_code", priority: 2}
+
+  # HumanEval
+  {description: "code:model=text_code,dataset=humaneval", priority: 1}
+
+  # LSAT
+  {description: "lsat_qa:model=text_code,task=all", priority: 2}
+  {description: "lsat_qa:model=text_code,task=grouping", priority: 3}
+  {description: "lsat_qa:model=text_code,task=ordering", priority: 3}
+  {description: "lsat_qa:model=text_code,task=assignment", priority: 3}
+  {description: "lsat_qa:model=text_code,task=miscellaneous", priority: 3}
+
+  # LegalSupport
+  {description: "legal_support:model=text_code", priority: 2}
+
+  # MedQA
+  {description: "med_qa:model=text_code", priority: 2}
+
+  # WMT
+  {description: "wmt_14:language_pair=en-cs,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-de,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-fr,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-hi,model=text", priority: 3}
+  {description: "wmt_14:language_pair=en-ru,model=text", priority: 3}
+  {description: "wmt_14:language_pair=cs-en,model=text", priority: 2}
+  {description: "wmt_14:language_pair=de-en,model=text", priority: 2}
```

Comment:
@percyliang How did you pick these language pairs? At the very least, these seem like very non-classical choices from MT literature for WMT-14.

(And I remain pretty firm reservations about including WMT. I think this just complicates the rankings in non-instructive ways. And this seems like it will also yield consternation from companies who say "oh we don't support lang ...")

(Also, idk how much we have stress-tested, but I worry about this breaking some tokenizers/etc. in some way due to non-robustness to non-English text. I realize this shouldn't happen if everything is robust, but given all of the time we have wasted dealing with tokenization, I would like to minimize said headaches.)

Reply:
This has not been stress tested in any way; it's plausible that tokenizers are an issue (e.g. CLEVA uses BLEU with a [different tokenizer](https://github.com/stanford-crfm/helm/blob/c545c4984dd3628bfb9577793609e7d7fdfd91d6/src/helm/benchmark/metrics/machine_translation_metrics.py#L71) than WMT)

Reply:
I think @YianZhang picked the languages...  Based on some experiments, it doesn't seem to break tokenization, etc.